### PR TITLE
Fix attribute passing to the mirror deleter

### PR DESCRIPTION
**Pull request by mraspaud**
The mirroring script was initialised wrongly as not information about the chain was passed on to the deleter.

This PR fixes this.

The problem originated in #20

### CODE REVIEW

File: bin/move_it_mirror.py [hunk 1]
```diff
@@ -20,59 +20,64 @@
...
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Move it Mirror."""
 
 import logging
 import logging.handlers
-import os
 from threading import Lock, Timer
-from six.moves.urllib.parse import urlparse, urlunparse
+from six.moves.urllib.parse import urlunparse
```
Comment (pnuu) [inline bin/move_it_mirror.py:28]: We don't support Python 2 anymore, so `six` isn't needed.

File: trollmoves/mirror.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""All you need for mirroring."""
+
+import os
+
+from six.moves.urllib.parse import urlparse
```
Comment (pnuu) [inline trollmoves/mirror.py:27]: We don't support Python 2 anymore, so `six` isn't needed.

File: trollmoves/mirror.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+file_registry = {}
+
+
+class MirrorRequestManager(RequestManager):
+    """Handle requests as a mirror."""
+
+    def __init__(self, port, attrs):
+        """Set up this mirror request manager."""
+        RequestManager.__init__(self, port, attrs)
+        self._deleter = MirrorDeleter(attrs)
```
Comment (pnuu) [inline trollmoves/mirror.py:42]: The addition of `attrs` to this call is the actual fix and was pretty well hidden among other changes, so I'll just highlight it here.

File: trollmoves/tests/test_server.py [hunk 4]
```diff
@@ -123,3 +124,12 @@ def test_process_notify(Message):
         with file_cache_lock:
             assert "/topic/20200428_1000_foo.tif" in file_cache
             assert len(file_cache) == 1
+
+
+class TestDeleter(unittest.TestCase):
```
Comment (pnuu) [inline trollmoves/tests/test_server.py:129]: Should there be a test that the file is actually deleted?

**Review by pnuu — APPROVED** [review]
Only some comments, LGTM.

File: trollmoves/tests/test_server.py (hunk 4 above)
Comment (mraspaud) [reply trollmoves/tests/test_server.py:129]: There should indeed, but I didn't change anything about this in the PR...

File: trollmoves/mirror.py (hunk 2 above)
Comment (mraspaud) [reply trollmoves/mirror.py:27]: thanks

**Pull request merged**